webapp/src/main/webapp/js/webconferencing.js

```diff
@@ -688,11 +688,11 @@
 		return initRequest(request);
 	};
 	
-	var getProvidersConfig = function() {
+	var getProvidersConfig = function(spaceIdentityId) {
```

Comment:
AdminApp.vue call this function, but, it in the admin app, we have no spaceIdentity.
So how it will work in this case ?

Reply:
> AdminApp.vue call this function, but, it in the admin app, we have no spaceIdentity. So how it will work in this case ?

In this case, the spaceIdentityId is transmitted as null, the space-specific case is not considered, resulting in the return of all providers.